### PR TITLE
Allow editing of cron expressions and notifications for CronJobs 

### PR DESCRIPTION
**Pull request by tvogels01**
@pettyjamesm @blahblahblah-  Please review.

### CODE REVIEW

File: bin [hunk 1]
```diff
@@ -0,0 +1 @@
+script
```
Comment (blahblahblah-) [inline bin]: Is this intended?

Comment (tvogels01) [inline bin]: Yes.  Looks weird in git but it's just a symbolic link to make the script directory available as bin.

File: app/controllers/jobs_controller.rb [hunk 2]
```diff
@@ -133,6 +133,6 @@ def verify_api_token
   private
 
   def job_params
-    params.require(:job).permit(:name, :notifications, :notification_ids, :buffer_time)
+    params.require(:job).permit(:name, :notifications, {:notification_ids => []}, :frequency, :cron_expression, :buffer_time)
```
Comment (blahblahblah-) [inline app/controllers/jobs_controller.rb]: Can you change it to `params.require(:job).permit(:name, :notifications, {notification_ids: []}, :frequency, :cron_expression, :buffer_time)` for Ruby 1.9 convention?

**Comment by blahblahblah-** [comment]
Besides the minor comment, looks good!

File: app/controllers/jobs_controller.rb (hunk 2 above)
Comment (tvogels01) [inline app/controllers/jobs_controller.rb]: Done.

**Pull request merged**